week-5-practise/js/main.js

```diff
@@ -0,0 +1,70 @@
+var blueBtn = document.querySelector("#blueBtn");
+
+var jumbotron = document.querySelector(".jumbotron");
+
+var donateAndVolunteerBtns = document.querySelectorAll(".btn-lrg");
+
+blueBtn.addEventListener("click", changeBlueColor);
+
+function changeBlueColor() {
+  jumbotron.style.backgroundColor = "#588fbd";
+
+  donateAndVolunteerBtns[0].style.backgroundColor = "#ffa500";
+
+  donateAndVolunteerBtns[1].style.backgroundColor = "black";
+
+  donateAndVolunteerBtns[1].style.color = "white";
+}
+
+var orangeBtn = document.querySelector("#orangeBtn");
+
+orangeBtn.addEventListener("click", changeOrangeColor);
+
+function changeOrangeColor() {
+  jumbotron.style.backgroundColor = "#f0ad4e";
+
+  donateAndVolunteerBtns[0].style.backgroundColor = "#5751fd";
+
+  donateAndVolunteerBtns[1].style.backgroundColor = "#31b0d5";
+
+  donateAndVolunteerBtns[1].style.color = "white";
+}
+var greenBtn = document.querySelector("#greenBtn");
+
+greenBtn.addEventListener("click", changeGreenColor);
+
+function changeGreenColor() {
+  jumbotron.style.backgroundColor = "#87ca8a";
+
+  donateAndVolunteerBtns[0].style.backgroundColor = "black";
+
+  donateAndVolunteerBtns[1].style.backgroundColor = "#8c9c08";
+}
+//PART-2
+var submitBttn = document.querySelector("button[type=submit]");
+
+submitBttn.addEventListener("click", submitInfo);
+
+function submitInfo(event) {
+  var valid = 3;
+  var emailInput = document.getElementById("exampleInputEmail1");
+  var textInput = document.getElementById("example-text-input");
+  var textAreaInput = document.getElementById("exampleTextarea");
```

Comment:
Instead of naming the `textInput` and `textAreaInput` references to the html inputs by the input *type*, try to name them instead for their MEANING.

Although the email input is a text input, you haven't called it `textInput1`, you've called it `emailInput` - that's great, because the reader knows what it's for, in terms of the business.

This will make it clearer to the reader what the inputs are for, and will also require fewer changes to your code if you happen to need to change the types of an input later on.


Can you do this for the other two variables which refer to the inputs?